images/builder/docker/custom-docker-builder/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 # The standard name for this image is openshift/origin-custom-docker-builder
 #
-FROM openshift/origin-base
+FROM openshift/origin-base:v1.5
```

Comment:
Hmm, i believe so. This is where part of problem comes for current images getting updated.